includes/admin/class-amp-template-customizer.php

```diff
@@ -252,15 +292,28 @@ public function add_customizer_scripts() {
 			true
 		);
 
+		$option_settings = [];
+		foreach ( $this->wp_customize->settings() as $setting ) {
+			/** @var WP_Customize_Setting $setting */
+			if ( 'option' === $setting->type ) {
+				$option_settings[] = $setting->id;
+			}
+		}
+
 		wp_add_inline_script(
 			'amp-customize-controls',
 			sprintf(
 				'ampCustomizeControls.boot( %s );',
 				wp_json_encode(
 					[
-						'queryVar' => amp_get_slug(),
-						'l10n'     => [
-							'ampVersionNotice' => __( 'You are customizing the AMP version of your site.', 'amp' ),
+						'queryVar'       => amp_get_slug(),
+						'optionSettings' => $option_settings,
+						'l10n'           => [
+							/* translators: placeholder is URL to non-AMP Customizer. */
+							'ampVersionNotice'     => wp_kses_post( sprintf( __( 'You are customizing the AMP version of your site. <a href="%s">Customize non-AMP version</a>.', 'amp' ), esc_url( admin_url( 'customize.php' ) ) ) ),
+							'optionSettingNotice'  => __( 'This also applies to the non-AMP version of your site.', 'amp' ),
```

Comment:
For a bit more conciseness may remove `This` ==> `Also applies to the non-AMP version of your site`